packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js

```diff
@@ -51,14 +51,12 @@ function getPageDetails( page ) {
 		} );
 	}
 
-	details.push( {
-		label: __( 'Parent' ),
-		// `null` indicates no parent.
-		value:
-			null === page?.parentTitle
-				? __( 'Top level' )
-				: decodeEntities( page?.parentTitle || __( '(no title)' ) ),
-	} );
+	if ( page?.parentTitle ) {
+		details.push( {
+			label: __( 'Parent' ),
+			value: decodeEntities( page?.parentTitle || __( '(no title)' ) ),
```

Comment:
```suggestion
			value: decodeEntities( page.parentTitle || __( '(no title)' ) ),
```

Nit: We already checked for the `undefined` above.

Reply:
Good catch!